ush/ufsda/yamltools.py

```diff
@@ -112,6 +144,36 @@ def fv3_geom_dict(case, levs, ntiles, layout, io_layout):
     return outdict
 
 
+def fv3land_geom_dict(case, levs, ntiles, layout, io_layout):
+    # returns a dictionary matching FV3-JEDI global geometry entries
+    outdict = {
+        'fms initialization': {
+            'namelist filename': '$(fv3jedi_fix_dir)/fmsmpp.nml',
+            'field table filename': '$(fv3jedi_fix_dir)/field_table',
+        },
+        'akbk': '$(fv3jedi_fix_dir)/akbk'+str(levs-1)+'.nc4',
+        'layout': layout,
+        'io_layout': io_layout,
+        'npx': str(case+1),
+        'npy': str(case+1),
+        'npz': str(levs-1),
+        'ntiles': str(ntiles),
+        'field metadata override': '$(fv3jedi_fieldmetadata_dir)/gfs-land.yaml',
+
+        'time invariant fields': {
+            'state fields': {
+                'datetime': '$(LAND_WINDOW_BEGIN)',
+                'filetype': 'fms restart',
+                'skip coupler file': 'true',
+                'state variables': '[orog_filt]',
+                'datapath': '/scratch2/BMC/gsienkf/Clara.Draper/data_RnR/orog_files_Mike/',
```

Comment:
We will need to stage these in a fix dir before merging it in, we will want to support both hera and orion

Reply:
Is it okay for the fix dir like `/scratch2/NCEPDEV/land/data/fix/` on hera?

Reply:
No, eventually we will want all the fix files with the other fix files, and these are maintained by the global-workflow team. This is fine for testing, but making a note here that this will have to be changed before we add land DA to the workflow itself.

Reply:
Add the comment line making a note to indicate future changes required before we add land DA to the workflow.

Reply:
This path was just intended as a temporary one for my tests while EMC finalized the fractional grids. I recommend changing this to the fix directory with the orography files that are being used by the rest of the work flow, since the land DA should be using the same files as the atmos DA.

Reply:
Also, do we need to change the orog file name to include the ocean details (mxXX) -  or again, just copy what's done on the atmos side.

Reply:
I think it's just the atm resolution for the orography but I don't know the details about the fractional grid files. I would just be consistent with what the coupled model is using on the atmos side if possible.

Reply:
The fractional grids have slightly different orography, and different files names. I think the main thing here is just to make sure that this is the same as for the atmos DA / and model.

Reply:
@jiaruidong2017 Can you please change this to use the same fix files as the model?

Reply:
@CoryMartin-NOAA Do you know where I can find the fix files for the atmos DA on hera? I  can do a check for the orography. Thanks.

Reply:
we don't use fix files (as of now) for orography for DA, but can we point to the model fix files that Kate Friedman et al. maintain? I assume that is what you want (what FV3 uses for orography). Or are the fields you need in the model RESTARTs?

Reply:
Thanks @CoryMartin-NOAA 
@barlage @KateFriedman-NOAA Do you know where we can find the FV3 model fix files for the variable `orography` maintained by the global workflow on hera? Thanks.

Reply:
@jiaruidong2017 The fix file set on Hera is here:

/scratch1/NCEPDEV/global/glopara/fix

The orography subset is:

/scratch1/NCEPDEV/global/glopara/fix/orog/20220805/

Note, the `20220805` subfolder is the current version for the orog files. When updates are made to those files a new timestamp (YYYYMMDD) subfolder will be created and users will be directed to point there for the orog files.

FYI, the global-workflow handles the fix file symlinks within GFS clones using a version file that a link script sources when setting up the clone for running. See these if curious:

https://github.com/NOAA-EMC/global-workflow/blob/develop/sorc/link_workflow.sh
https://github.com/NOAA-EMC/global-workflow/blob/develop/versions/fix.ver

Reply:
@KateFriedman-NOAA Thank you very much for your information.